### PR TITLE
Use pi-ui's Checkbox instead of custom in `Set Mixed Account` page

### LINKED ISSUE #3049: Use pi-ui's Checkbox instead of custom in `Set Mixed Account` page

### PR DESCRIPTION
**Pull request by bgptr**
closes #3049

### CODE REVIEW

File: app/components/views/GetStartedPage/SetMixedAcctPage/SetMixedAcctPage.jsx [hunk 1]
```diff
@@ -116,32 +116,20 @@ export default ({ onSendBack, onSendContinue }) => {
...
                         id={"change"+acctIdx}
-                        name={"a"+acctIdx}
-                        type="checkbox"
+                        label={<T id="getstarted.setAccount.change" m="Set Unmixed Account" />}
                         checked={changeAcctIdx === acctIdx}
                         onChange={() => onSubmitSetChange(acctIdx)}
-                        value={acctIdx}
                       />
-                      <label htmlFor={"change"+acctIdx} className={styles.checkboxLabel}></label>
-                      <div className={styles.label}>
```
Comment (amassarwi) [inline app/components/views/GetStartedPage/SetMixedAcctPage/SetMixedAcctPage.jsx:142]: Please ditch `.label` from .css

**Review by amassarwi — COMMENTED** [review]
LGTM, Added one comment.

Probably should wait on #3033.

File: app/components/views/GetStartedPage/SetMixedAcctPage/SetMixedAcctPage.jsx (hunk 1 above)
Comment (bgptr) [reply app/components/views/GetStartedPage/SetMixedAcctPage/SetMixedAcctPage.jsx:142]: Thank you for the review @amassarwi 
I tried to ditch `.label` as well as the `checkboxLabel`, but I did not find it in SetMixedAcctPage.module.css  :) Could you please help me find it?

Comment (amassarwi) [reply app/components/views/GetStartedPage/SetMixedAcctPage/SetMixedAcctPage.jsx:142]: oh i see, then probably nothing to do here

**Comment by jholdstock** [comment]
Agreed it would be good to get #3033 merged before this one goes in. Looks good though!

**Review by vctt94 — APPROVED** [review]
LGTM. Are we getting 3033 in before release? This one is pretty small, I think we can fit it

**Comment by alexlyp** [comment]
rebase please

**Comment by bgptr** [comment]
@alexlyp rebased

**Review by alexlyp — APPROVED** [review]

**Pull request merged**